### PR TITLE
Readme fix #1

### PR DESCRIPTION
**Pull request by episyrphus**

### CODE REVIEW

**Comment by episyrphus** [comment]
bro can you even see this?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # Gronti
 Lol wtf are we doin tho
+
+We are doing GitHub Greg.
```
Comment (gfalbery) [inline README.md]: This is good but it needs a comma

Comment (gfalbery) [inline README.md]: Also ur a fuckin idiot lol

Comment (gfalbery) [inline README.md]: Sorry that was mean it's bin a long day

Comment (gfalbery) [inline README.md]: image

Comment (gfalbery) [inline README.md]: http://i.imgur.com/XryBYTI.gif

**Pull request merged**